libclc/generic/include/config.h

```diff
@@ -20,6 +20,8 @@
  * THE SOFTWARE.
  */
 
+#include <func.h>
```

Comment:
```suggestion
#include "func.h"
```
This header is located in the same directory, so we can make it work w/o setting additional include directory paths.

Reply:
Good point, suggestion applied